src/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
Why not just add `src/bin/` to the root `.gitignore`?

Reply:
It's already there. This file was auto-generated by Eclipse, and I committed it by mistake. Removed. Good catch.

Reply:
It will run under Oracle Java 8, Openjdk 8, and Openjdk 11. Unfortunately Bazel requires Java 8 so you'll need either Oracle Java 8 or Openjdk 8. I recommend Adoptopenjdk if you want to use the latter: https://github.com/AdoptOpenJDK/homebrew-openjdk